.babelrc

```diff
@@ -6,14 +6,30 @@
   "env": {
     "coverage": {
       "plugins": [
-        ["istanbul"]
+        [
+          "flow-react-proptypes",
+          "transform-flow-strip-types",
+          "istanbul"
+        ]
+      ]
+    },
+    "development": {
+      "sourceMaps": "both",
+      "plugins": [
+        "flow-react-proptypes",
```

Comment:
> Why would we need to generate in production just to remove them?

Tricky isn't it. The thing is that don't just remove them, we wrap the definitions around a *is production* conditional clause. That means user can have warnings in dev and a lighter build in production.